src/pages/Category/Pagination.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import styled from 'styled-components';
+
+function Pagination(props) {
+  const postsPerPage = props.postsPerPage;
+  const cardDataLength = props.cardDataLength;
+  const pageNumbers = [];
+  for (let i = 1; i <= Math.ceil(cardDataLength / postsPerPage); i++) {
+    pageNumbers.push(i);
+  }
```

Comment:
실제 사용되는 부분을 봤는데 만약 pageNumbers 배열만 사용한다면 postsPerPage, cartDataLength를 props로 받는게 아니라
pageNumber만 props로 받는게 좋아 보입니다!
그리고 for문 보다는 map, filter, forEach등의 메소드를 적극적으로 활용하시는게(선언적 프로그래밍 방법을 활용) 가독성측면에서 좋습니다!

```suggestion
function Pagination({pageNumber}) {
  const pageNumberList = new Array(pageNumber).fill(1).map((el,idx) => idx+1)
```

Reply:
어케저케 적용은 했는데.. 코드가 10줄은 확 줄었어요.. 얼마나 해야 멘토님의 경지까지 올라갈 수 있을까요.... 존경존경 👍👍👍👍👍  또 하나 배웠습니다!!!(까먹지 말아야 할 텐데 말이죠...ㅠㅠ)

Reply:
너무 과찬해주시니 부끄럽네요... 저도 잘 못합니다 어서 동료개발자되셔서 같이 공부하면서 열심히 해봐요

Reply:
동료개발자..란 단어가 뭉클하네요.ㅎㅎㅎ 위코드 생활 중 어제가 가장.. 안풀리는 날인것 같았어요.ㅎㅎ 열심히.. 더 열심히.. ㅎㅎ